### PR TITLE
(bugfix) Client Redirect from '/' to '/home'

### PR DESCRIPTION
**Pull request by ZacButko**
## Requirements
- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.

#### For changes to apps
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
(N/A)

#### If applicable
- [ ] My work includes tests or is validated by existing tests.
- [ ] I have updated the [esm-framework mock](https://github.com/openmrs/openmrs-esm-core/blob/master/packages/framework/esm-framework/mock.tsx) to reflect any API changes I have made.
(N/A)

## Summary
Solves ticket [OS-782]( https://issues.openmrs.org/browse/O3-782) from Client side
This is an additional safeguard for serverless setups. Primary fix done by @ibacher [here](https://github.com/openmrs/openmrs-contrib-ansible-docker-compose/commit/fb39323ab29b70d2c575fa0786165b067a13dc12)

## Screenshots
No UI Changes

## Related Issue
[OS-782]( https://issues.openmrs.org/browse/O3-782)

### CODE REVIEW

File: packages/apps/esm-primary-navigation-app/src/index.ts [hunk 1]
```diff
@@ -42,6 +44,14 @@ function setupOpenMRS() {
         offline: true,
         order: 0,
       },
+      {
+        load: getSyncLifecycle(() => HomeRedirect(), options),
```
Comment (brandones) [inline packages/apps/esm-primary-navigation-app/src/index.ts:48]: I think you could actually just write the single-spa lifecycle object, something like

```ts
load: {
  bootstrap: () => { navigateToUrl(window.getOpenmrsSpaBase() + "home"); },
  mount: () => {},
  unmount: () => {}
}
```

File: packages/apps/esm-primary-navigation-app/src/index.ts [hunk 2]
```diff
@@ -42,6 +44,14 @@ function setupOpenMRS() {
...
+        route: (location: Location) =>
+          location.pathname === window.getOpenmrsSpaBase(),
+        online: true,
+        offline: true,
+        order: 0,
```
Comment (brandones) [inline packages/apps/esm-primary-navigation-app/src/index.ts:53]: This doesn't have meaning here, and is also the default value

File: packages/apps/esm-primary-navigation-app/src/index.ts (hunk 1 above)
Comment (ZacButko) [reply packages/apps/esm-primary-navigation-app/src/index.ts]: pretty close. These need to be promises

File: packages/apps/esm-primary-navigation-app/src/index.ts (hunk 2 above)
Comment (ZacButko) [reply packages/apps/esm-primary-navigation-app/src/index.ts:56]: Order apparently is necessary, otherwise it crashes 🙄

Comment (brandones) [reply packages/apps/esm-primary-navigation-app/src/index.ts:55]: Oh really??? That's a bug

**Pull request merged**